src/NHibernate/Linq/Visitors/NhPartialEvaluatingExpressionVisitor.cs

```diff
@@ -81,7 +81,15 @@ public override Expression Visit(Expression expression)
 				#region NH additions
 				// Variables should be evaluated only when they are part of an evaluatable expression (e.g. o => string.Format("...", variable))
 				expression is UnaryExpression unaryExpression &&
-				ExpressionsHelper.IsVariable(unaryExpression.Operand, out _, out _))
+				(
```

Comment:
Can the expression inside parenthesis please be moved to a method/methods with a meaningful name?

Remember, if something requires a comment it means it should go to a method.

Reply:
Done.